backend/src/main/java/com/github/EazyErik/service/CustomDrinksService.java

```diff
@@ -93,4 +94,23 @@ public List<CustomDrink> getDrinksByIngredient(String ingredient) {
                 .toList();
 
     }
+
+    public List<CustomDrink>  getRandomCocktail() {
```

Comment:
Why does `getRandomCocktail` (singular) return a list of cocktails. Since the list is either empty or contains exactly one element you can use `Optional<CustomDrink>`.